newm-server/src/main/kotlin/io/newm/server/database/migration/V51__WalletConnectionsUpdates.kt

```diff
@@ -0,0 +1,14 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V51__WalletConnectionsUpdates : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            exec("ALTER TABLE wallet_connection_challenges DROP COLUMN payload")
```

Comment:
And an `if exists` in here just for extra safety.

Reply:
Done